### PR TITLE
deps(iced): upgrade iced to 0.12

### PR DESCRIPTION
**Pull request by edgy-b**
- upgraded to iced 0.12
- fixed white theme dropdown selector color

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "uad-ng"
 description = "A cross-platform GUI debloater for android devices"
-version = "1.0.2"
+version = "1.1.0"
```
Comment (Frigyes06) [inline Cargo.toml:4]: Please don't bump the UAD version.

File: src/core/theme.rs [hunk 2]
```diff
@@ -65,7 +65,7 @@ impl Theme {
                     foreground: color!(0xE0E0E0),
                 },
                 normal: NormalColors {
-                    primary: color!(0x230F08),
+                    primary: color!(0x818181),
```
Comment (Frigyes06) [inline src/core/theme.rs:68]: Why are you changing the color here?

File: src/gui/mod.rs [hunk 3]
```diff
@@ -369,13 +369,16 @@ impl UadGui {
 
         Self::run(Settings {
             window: Window {
-                size: (820, 640),
+                size: iced::Size {
+                    width: 1000.0,
```
Comment (Frigyes06) [inline src/gui/mod.rs:373]: Why are you changing the startup size?

**Review by Frigyes06 — CHANGES_REQUESTED** [review]

**Comment by Frigyes06** [comment]
Fixes #326

File: src/gui/mod.rs (hunk 3 above)
Comment (edgy-b) [reply src/gui/mod.rs:373]: Let's say there is update: buttons get pushed and some are not visible
old-res
VS new:
new-res

File: src/core/theme.rs (hunk 2 above)
Comment (edgy-b) [reply src/core/theme.rs:68]: To fix dropdown on Light theme.
Old:
old-drop
New:
new-drop

File: Cargo.toml (hunk 1 above)
Comment (edgy-b) [reply Cargo.toml:4]: Ok, will revert back.

**Comment by AnonymousWP** [comment]
> Fixes #326

This is not how it works. You either need to put it in the description of the author that opened the PR, or link it here:

image

I've done it now.

**Comment by AnonymousWP** [comment]
> * upgraded to iced 0.12
> * fixed white theme dropdown selector color

In my opinion the second thing should have been done in a separate PR.

**Comment by AnonymousWP** [comment]
What are you trying to do exactly? You keep updating this branch without retaining a clean commit history.

**Comment by edgy-b** [comment]
> What are you trying to do exactly? You keep updating this branch without retaining a clean commit history.

Right👍 I was playing with some bash scripts, crons etc. automation locally, not a good idea on PR branches

**Comment by AnonymousWP** [comment]
Might as well update to iced 0.12.1 as it comes with important fixes: https://github.com/iced-rs/iced/releases/tag/0.12.1

**Comment by edgy-b** [comment]
> Might as well update to iced 0.12.1 as it comes with important fixes: https://github.com/iced-rs/iced/releases/tag/0.12.1

It is indeed coming with 0.12.1 in Cargo.lock

**Comment by AnonymousWP** [comment]
> > Might as well update to iced 0.12.1 as it comes with important fixes: https://github.com/iced-rs/iced/releases/tag/0.12.1
> 
> It is indeed coming with 0.12.1 in Cargo.lock

What do you mean? In your changes I see 0.12.0, not 0.12.1.

**Comment by edgy-b** [comment]
> > > Might as well update to iced 0.12.1 as it comes with important fixes: https://github.com/iced-rs/iced/releases/tag/0.12.1
> > 
> > 
> > It is indeed coming with 0.12.1 in Cargo.lock
> 
> What do you mean? In your changes I see 0.12.0, not 0.12.1.
https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/331/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eR1168

File: src/gui/mod.rs (hunk 3 above)
Comment (adhirajsinghchauhan) [reply src/gui/mod.rs:373]: Once #340 is merged, you'd need to revisit this to adjust width accordingly.

**Review by Frigyes06 — APPROVED** [review]

**Pull request merged**